portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,7 +58,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     // Add new comment only if the string is more than just white spaces
     if(text.trim().length() > 0){
-      comments.add(text);
+      long timestamp = System.currentTimeMillis();
+
+      Entity taskEntity = new Entity("Comment");
+      taskEntity.setProperty("text", text);
+      taskEntity.setProperty("timestamp", timestamp);
```

Comment:
What do you think about calling this timestamp_ms or timestampMs to make it clear that these are in milliseconds?